sources/packages/web/src/composables/useInstitutionStore.ts

```diff
@@ -0,0 +1,13 @@
+import { Store, useStore } from "vuex";
```

Comment:
There is already a composable for institution store. You can add the new composable method here. Also for the method, I would recommend naming it getLocationName instead of locationName.

![image](https://user-images.githubusercontent.com/54600590/186505223-e5748766-d46b-49a1-87ff-6925b1ec684b.png)

You can rename this composable to useInstitutionStore and update its dependencies.(this is up to you)